src/models/OtrStore.js

```diff
@@ -24,6 +24,9 @@ function OtrStore(name, client) {
 OtrStore.prototype.initChan = function(args) {
 	var network_id = args.network;
 	var chan = args.chan;
+	var otrStore = this;
+	var client = this.client;
+
 	if (chan.type === Chan.Type.QUERY) {
 		var network = _.find(this.client.networks, {id: network_id});
```

Comment:
(btw I know style inconsistency hurts your eyes & brain, but this is not related to this PR diff ;-)